### ISSUE TITLE
Wrapping class swig mapping issue

### ISSUE THREAD

**Issue by askuric** [pyomeca/biorbd#222]
Hi, first of all, thanks a lot for the great library!

I've been using it for calculating the achievable force polytopes of a human arm, at the moment I'm working in python and I'm trying to translate the opensim models to the biorbd format. So, I'll need to include the wrapping of muscles in my models, and this seems to be well supported by the biorbd, which is great. :smile: 

But when the wrapping is enabled and if I try to visualize the model, using the bioviz, or just run the example provided in the libray I've got some issues of `TypeError` related to swig I think. 
``` sh
$ python Test_longueur_wrapping.py 
Traceback (most recent call last):
  File "/home/user/github/pyomeca/biorbd/examples/Test_longueur_wrapping.py", line 39, in <module>
    RTw = biorbd.WrappingHalfCylinder(model.muscle(0).pathModifier().object(0)).RT(
  File "/home/user/anaconda2/envs/pyomeca/lib/python3.9/site-packages/biorbd/biorbd.py", line 5855, in RT
    return _biorbd_eigen.WrappingHalfCylinder_RT(self, model, Q, updateKin)
TypeError: Wrong number or type of arguments for overloaded function 'WrappingHalfCylinder_RT'.
  Possible C/C++ prototypes are:
    BiorbdEigen3::muscles::WrappingHalfCylinder::RT(BiorbdEigen3::rigidbody::Joints &,BiorbdEigen3::rigidbody::GeneralizedCoordinates const &,bool)
    BiorbdEigen3::muscles::WrappingHalfCylinder::RT(BiorbdEigen3::rigidbody::Joints &,BiorbdEigen3::rigidbody::GeneralizedCoordinates const &)
```
I get the same error if I try to calculate the radius or lenght:
```sh
Traceback (most recent call last):
  File "/home/user/github/pyomeca/biorbd/examples/Test_longueur_wrapping.py", line 39, in <module>
    l = biorbd.WrappingHalfCylinder(model.muscle(0).pathModifier().object(0)).length()
  File "/home/user/anaconda2/envs/pyomeca/lib/python3.9/site-packages/biorbd/biorbd.py", line 5870, in length
    return _biorbd_eigen.WrappingHalfCylinder_length(self)
TypeError: in method 'WrappingHalfCylinder_length', argument 1 of type 'BiorbdEigen3::muscles::WrappingHalfCylinder const *'
```

Would you suggest me to build the project from source to avoid this error?
For now I've got the conda version of the biorbd and bioviz.
I've got swig `SWIG Version 4.0.2` and python `Python 3.9.5`

**Comment by pariterre** [comment]
That is correct, I just noticed that the downcasting for halfcylinder is not implemented! 
I will push the donwcasting capability in your PR!
Sorry about that!

**Comment by pariterre** [comment]
Fixed in #221 
Thanks for reporting!
